eventservice/api/categories.json

```diff
@@ -1,7 +1,15 @@
-{"category":"Entertainment"}
-{"category":"Academic"}
-{"category":"Community"}
+{"category":"Campus Visits"}
 {"category":"Career Development"}
-{"category":"Recreation"}
-{"category":"Athletics","subcategories":["Baseball","Men's Basketball","Men's Cross Country","Football","Men's Golf","Men's Gymnastics","Men's Tennis","Men's Track Field","Wrestling","Women's Basketball","Women's Cross Country","Women's Golf","Women's Gymnastics","Women's Soccer","Softball","Swim Dive","Women's Tennis","Women's Track Field","Volleyball"]}
+{"category":"Ceremonies & Services"}
+{"category":"Club Athletics"}
+{"category":"Conferences & Workshops"}
+{"category":"Exhibits"}
+{"category":"Festivals & Celebrations"}
+{"category":"Film Screenings"}
+{"category":"Performances"}
+{"category":"Receptions & Open House Events"}
+{"category":"Social & Informal Events"}
+{"category":"Speakers & Seminars"}
+{"category":"Big 10 Athletics","subcategories":["Baseball","Men's Basketball","Men's Cross Country","Football","Men's Golf","Men's Gymnastics","Men's Tennis","Men's Track Field","Wrestling","Women's Basketball","Women's Cross Country","Women's Golf","Women's Gymnastics","Women's Soccer","Softball","Swim Dive","Women's Tennis","Women's Track Field","Volleyball"]}
+{"category":"Recreation, Health & Fitness","subcategories":["Group Fitness","Aquatics","Ice Skating","Personal Training","Student Wellness","Adventure Rec","Summer Camp"]}
 {"category":"Other"}
```

Comment:
I compared this with the feature description. The "Other" category needs to be removed. Thanks.